pixi.toml

```diff
@@ -158,13 +158,13 @@ blas-devel = "*"
 # XXX: when updating this task, remember to update other build tasks if appropriate
 [feature.build-task.tasks.build]
 cmd = "spin build --setup-args=-Dblas=blas --setup-args=-Dlapack=lapack --setup-args=-Duse-g77-abi=true"
-cwd = "../.."
+cwd = "."
```

Comment:
I think this is the default and can hence be left out? Saves a bunch of boilerplate lines.